AlastriaID/VerifiableAttestation/2022-11/schema.json

```diff
@@ -13,149 +13,163 @@
       }
     },
     "id": {
-      "description": "Defines unique identifier of the Verifiable Attestation",
+      "description": "Property. Defines unique identifier of the Verifiable Attestation",
       "type": "string",
       "format": "uri"
     },
     "type": {
-      "description": "Defines the Verifiable Credential type",
+      "description": "Property. Defines the Verifiable Credential type",
       "type": "array",
       "items": {
         "type": "string"
       }
     },
     "issuer": {
-      "description": "Defines the issuer of the Verifiable Attestation",
+      "description": "Property. Defines the issuer of the Verifiable Attestation",
       "type": "string",
       "format": "uri"
     },
     "issuanceDate": {
-      "description": "Defines the date and time, when the Verifiable Attestation becomes valid",
+      "description": "Property. Defines the date and time, when the Verifiable Attestation becomes valid",
       "type": "string",
       "format": "date-time"
     },
     "issued": {
```

Comment:
@alejandro-nieto-git This value of the example.json has been removed because it is deprecated, why is it kept in the schema?

Reply:
Well it is deprecated but EBSI left it in the schema and example but I think we must remove it from both. I will update the PR. Thanks!